### ISSUE TITLE
ckanext/datastore/templates/ajax_snippets/api_info.html does not links in api examples

### ISSUE THREAD

**Issue by ebuckley** [ckan/ckan#4304]
### CKAN Version if known (or site URL)
Ckan 2.7 commit (bc4f8dfc0d60c51858d82cd19c181f49ae8cf21e)

### Please describe the expected behaviour
When viewing a dataset resource which has been loaded with the datastore
And I click on the green "Data API" button
I expect to see usable examples for accessing that resource

### Please describe the actual behaviour
When I click the "Data API" button, I see the right content, but the links are not wrapped in anchor tags.

In CKAN 2.6, the links are click-able. This seems to be a smoother user experience for users that are motivated to explore and use the resources API.

### What steps can be taken to reproduce the issue? 
Install CKAN 2.7 and enable the datastore plugin.

### TLDR

- The `ckanext/datastore/templates/ajax_snippets/api_info.html` template does not display links for the example api queries.
- In CKAN 2.6 the example api calls were links.
- If having links is desirable, then I would be interested in contributing a PR

**Comment by wardi** [comment]
@ebuckley yes, we removed the links because of excessive bot traffic on some large sites, but a better fix might have been to add `rel="nofollow"` to those links. A PR would be great!
